bridges/snowbridge/parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -185,7 +185,7 @@ where
 			DescendOrigin(PalletInstance(inbound_queue_pallet_index).into()),
 			// Change origin to the bridge.
 			UniversalOrigin(GlobalConsensus(network)),
-			// Call create_asset on foreign assets pallet.
+			// Call create_asset on foreign assets' pallet.
```

Comment:
```suggestion
			// Call create_asset on the foreign assets pallet.
```

"foreign assets" is the identification of the pallet